.gitattributes

```diff
@@ -0,0 +1,2 @@
+/current/** linguist-generated
```

Comment:
Out of interest: 

What is the motivation of this? Discounting XML files from language statistics? Hiding XML files in the diffs?

Reply:
Hiding XML files in the diffs but they can still be shown. E.g. you can compare collapsed: https://github.com/cceh/papyri-wl-data/compare/master...creativeDev6:master with uncollapsed: https://github.com/cceh/papyri-wl-data/commit/39132135ed795df796c95409f5c79dabb10697f8 by scrolling down to the several xml files, e.g. current/grc/general/wl-grc-00001.xml. In my opinion this makes it more clean since they are generated. The XML files are also not listed in the language stastics bar on the main repo page. The only problem could be they will not be listed here https://github.com/cceh/papyri-wl-data/search?l=XML anymore. Unfortunately, I cannot see this on my fork https://github.com/creativeDev6/papyri-wl-data/search?l=XML.

After all it might be better to remove `.gitattributes`. In https://github.com/cceh/papyri-wl-data/pull/20/commits/c9ecbe0860e444cafebd6029df137b35a0727609 I also commited several files within the staging folders but they will not be shown because you have to scroll through all the XML files first. This was also a reason for collapsing the XML files. However, there seems to be a cap to display only around 3000 files so the important files within the staging folder are not shown at all. So for next time I should split the the commits into the generated files from current folder and the important changed files that will also affect the generated files.

@pdaengeli I think I will leave `.gitattributes` for the moment and see what kind of change it will be making towards the language statistics and remove it if it has any bad consequences.

Reply:
Thanks for the explanations. I don't mind `.gitattributes` in the repo. And XML doesn't need to be included in the language statistic or search results.

Splitting commits between changed logic/metadata and generated files is a good thing. In any case it is always possible to append `.diff` to the commit URL to bypass capping (this will return the full diff in plain text, not rendered).